src/HasSlug.php

```diff
@@ -122,6 +122,21 @@ protected function getSlugSourceString(): string
 
         $slugSourceString = collect($this->slugOptions->generateSlugFrom)
             ->map(function (string $fieldName) : string {
+
+                if (strpos($fieldName, '.') !== false) {
```

Comment:
Could you extract this bit to it's own method. Bonus points for not having to use `continue`.

Reply:
ok :)